byterun/caml/misc.h

```diff
@@ -40,13 +40,22 @@ typedef char * addr;
 
 #ifdef __GNUC__
   /* Works only in GCC 2.5 and later */
-  #define Noreturn __attribute__ ((noreturn))
+  #define CAMLnoreturn __attribute__ ((noreturn))
 #elif _MSC_VER >= 1500
-  #define Noreturn __declspec(noreturn)
+  #define CAMLnoreturn __declspec(noreturn)
 #else
+  #define CAMLnoreturn
+#endif
+
+#ifdef __GNUC__
+  /* Works only in GCC 2.5 and later */
+  #define Noreturn __attribute__ ((noreturn))
+#elif 
```

Comment:
Travis [rightly complains](https://travis-ci.org/ocaml/ocaml/builds/72992638) about this line.